From: Thorsten Kukuk <kukuk@thkukuk.de>
Date: Mon, 4 Apr 2016 13:37:32 +0000 (-0400)
Subject: getpublickey.c: ifdef out yp headers
X-Git-Tag: libtirpc-1-0-2-rc3~3
X-Git-Url: http://git.linux-nfs.org/?p=steved%2Flibtirpc.git;a=commitdiff_plain;h=742bbdff6ddff4dde0d610a842cd8ac0408af0a0

getpublickey.c: ifdef out yp headers

If we don't compile in YP support, don't include YP

Signed-off-by: Thorsten Kukuk <kukuk@thkukuk.de>
Signed-off-by: Steve Dickson <steved@redhat.com>
---

diff --git a/src/getpublickey.c b/src/getpublickey.c
index 764a5f9..8cf4dc2 100644
--- a/src/getpublickey.c
+++ b/src/getpublickey.c
@@ -38,8 +38,10 @@
 #include <pwd.h>
 #include <rpc/rpc.h>
 #include <rpc/key_prot.h>
+#ifdef YP
 #include <rpcsvc/yp_prot.h>
 #include <rpcsvc/ypclnt.h>
+#endif
 #include <string.h>
 #include <stdlib.h>
 

